corehq/apps/hqwebapp/crispy.py

```diff
@@ -319,3 +319,9 @@ class B3HiddenFieldWithErrors(Field):
 
 class RadioSelect(Field):
     template = "hqwebapp/crispy/radioselect.html"
+
+
+def make_form_readonly(form):
```

Comment:
Doesn't look like this was ever used.

Reply:
Nevermind, I must have done ctrl+F without expanding the diffs that don't display by default because they're large.